src/app/global/local_account_sensitive_settings.nim

```diff
@@ -805,6 +809,30 @@ QtObject:
     write = setStickersEnsRopsten
     notify = stickersEnsRopstenChanged
 
+  proc isDDMMYYDateFormatChanged*(self: LocalAccountSensitiveSettings) {.signal.}
```

Comment:
is it not possible instead to receive dateFormat. instead of isDDMMYY or is@3Hr? wouldn't that make more sense, and we wouldn't have to add more properties in case we have a different format in future?

Reply:
I was thinking on maybe adding an enumeration but then thought that maybe it doesn't make sense because it is just adding more complexity now and maybe is not used never since the "normal" options are just only 2 in each case, 12h or 24h, and british or american format for dates. If there are other options in the future they will be custom ones and we should update not just only this, also the calls in the view and the view itself. But if you really think it will benefit in somehow to change it in a general way, yes, I can do the change.

Reply:
I just feel these tow a re mutually exclusive if 1 is false the other will be true and maybe handling it with two properties is not the best? should we pass it as a string? now sure if enums can be exposed in NIM? and we can add constants to the Enum on the qml side and check that instead?

Reply:
Not sure about exposing them as strings so we can create a task to address it by using enums when C++ app is prepared. Here the new created [task](https://github.com/status-im/status-desktop/issues/5635).